pycbc/workflow/grb_utils.py

```diff
@@ -471,7 +471,7 @@ def build_segment_filelist(seg_dir):
 def make_pygrb_plot(workflow, exec_name, out_dir,
                     ifo=None, inj_file=None, trig_file=None,
                     onsource_file=None, bank_file=None,
-                    seg_files=None, tags=None):
+                    seg_files=None, tags=None, **kwargs):
```

Comment:
`**kwargs` should only be used if this function does not know what kwargs are being given. `plot_bkgd` is sent and ingested by this function, so should be given explicitly.